### PR TITLE
[daftsex] Fix yt-dlp#2637

### PR DESCRIPTION
**Pull request by Soebb**
## Please follow the guide below

- You will be asked some questions, please read them **carefully** and answer honestly
- Put an `x` into all the boxes [ ] relevant to your *pull request* (like that [x])
- Use *Preview* tab to see how your *pull request* will actually look like

---

### Before submitting a *pull request* make sure you have:
- [x] At least skimmed through [contributing guidelines](https://github.com/yt-dlp/yt-dlp/blob/master/CONTRIBUTING.md#developer-instructions) including [yt-dlp coding conventions](https://github.com/yt-dlp/yt-dlp/blob/master/CONTRIBUTING.md#yt-dlp-coding-conventions)
- [x] [Searched](https://github.com/yt-dlp/yt-dlp/search?q=is%3Apr&type=Issues) the bugtracker for similar pull requests
- [x] Checked the code with [flake8](https://pypi.python.org/pypi/flake8)

### In order to be accepted and merged into yt-dlp each piece of code must be in public domain or released under [Unlicense](http://unlicense.org/). Check one of the following options:
- [x] I am the original author of this code and I am willing to release it under [Unlicense](http://unlicense.org/)
- [ ] I am not the original author of this code but it is in public domain or released under [Unlicense](http://unlicense.org/) (provide reliable evidence)

### What is the purpose of your *pull request*?
- [x] Bug fix
- [ ] Improvement
- [ ] New extractor
- [ ] New feature

---

### Description of your *pull request* and other information
Fix daftsex extractor

Explanation of your *pull request* in arbitrary form goes here. Please make sure the description explains the purpose and effect of your *pull request* and is worded well enough to be understood. Provide as much context and examples as possible.

### CODE REVIEW

**Comment by pukkandan** [comment]
What's with the massive try block?

**Comment by Soebb** [comment]
Please do desired changes yourself if you can.

**Comment by Soebb** [comment]
> What's with the massive try block? It's not clear from which lines the errors are expected to be from

Sorry about that, can you check again?

File: yt_dlp/extractor/daftsex.py [hunk 1]
```diff
@@ -52,28 +68,79 @@ def _real_extract(self, url):
...
+                'id': video_id,
+                'title': title,
+                'formats': formats,
+                'comment_count': int_or_none(item.get('comments')),
+                'description': description,
+                'duration': duration,
+                'thumbnails': thumbnails,
+                'timestamp': timestamp,
+                'view_count': views,
+            }
```
Comment (pukkandan) [inline yt_dlp/extractor/daftsex.py:146]: Since the if block returns, else can be removed and the whole thing dedented

**Review by pukkandan — CHANGES_REQUESTED** [review]
also modify both tests so they pass

**Pull request merged**